README.md

```diff
@@ -36,6 +36,7 @@ git pull|Pull down any changes that were checked into a branch (Git repository).
 git push|Push your committed changes to your currently checked out branch.
 git push origin master|Push your committed changes up to your master branch. 'master' may be subsituted for another branch.
 git status|Get the status of files in the current repo.
+git status -uno|Check if your local branch is up-to-date with the remote one
```

Comment:
@freeranger I would not recommend "git status -uno" because if you add a new file then this will not get picked up in the status change. "git status -uno" is the same as:

git status --untracked-files=no

Or:

git config status.showuntrackedfiles no

This is not good! Typically when you run status you want to see any new untracked files that need to be added. I would recommend just running a simple "git status" as this will reveal whether or not your local branch is up to date with the remote.